src/gcc_pkgconfig.mk

```diff
@@ -3,7 +3,11 @@
 
 F77 = gfortran
 
+F77DEBUGFLAGS= -ffpe-trap=invalid,zero,overflow -fbounds-check
 F77FLAGS=-DVERSION=\"$(VERSION)\" -O2 -ftree-vectorize -fno-math-errno -g -mavx2 -mfma -Wall -Wextra -fimplicit-none -fmodule-private -Wno-conversion
+ifdef DEBUG
```

Comment:
This flag should have a more suggestive name, DEBUG does not describe much at all. How about `SNAP_FPE_CHECK` and `SNAP_BOUNDS_CHECK`? This would convey that these checks are not free